src/plugins/world_control/WorldControl.qml

```diff
@@ -82,6 +92,24 @@ RowLayout {
    */
   property bool paused: false
 
+  /**
+   * Reset
+   */
+  RoundButton {
```

Comment:
Can we add a hovered tooltip that states what reset does?

Reply:
https://github.com/gazebosim/gz-gui/pull/476/commits/639643dc602fa71686806d2188548c14f254461b